test/nn/test_load_state_dict.py

```diff
@@ -362,7 +365,162 @@ def test_load_state_dict_warn_assign(self):
             m.load_state_dict(state_dict)
 
 
+class MyLoadTensor(torch.Tensor):
+
+    # enable deepcopy
+    def new_empty(self, shape):
+        return type(self)(shape)
+
+    @classmethod
+    def __torch_function__(cls, func, types, args=(), kwargs=None):
+        kwargs = {} if kwargs is None else kwargs
+
+        def module_load_from(dest, src):
+            # always convert src to MyLoadTensor
+            if type(src) is torch.Tensor:
+                return MyLoadTensor(src)
+            elif type(src) is MyLoadTensor:
+                return src
+            elif type(src) is MyWrapperLoadTensor:
+                return MyLoadTensor(src._data)
```

Comment:
What is `._data` ??

Reply:
`MyWrapperLoadTensor._data`, I named the attribute of the wrapper subclass as such, but can rename

Reply:
Ho my bad. I though it was a field on the base Tensor.